src/openrct2/windows/dropdown.c

```diff
@@ -174,8 +174,15 @@ void window_dropdown_show_text_custom_width(sint32 x, sint32 y, sint32 extray, u
     _dropdown_item_width = width;
     _dropdown_item_height = (flags & DROPDOWN_FLAG_CUSTOM_HEIGHT) ? custom_height : 10;
     gDropdownNumItems = (sint32)num_items;
-    _dropdown_num_columns = (gDropdownNumItems + DROPDOWN_TEXT_MAX_ROWS - 1) / DROPDOWN_TEXT_MAX_ROWS;
-    _dropdown_num_rows = (gDropdownNumItems + _dropdown_num_columns - 1) / _dropdown_num_columns;
+    // There must always be at least one column to prevent dividing by zero
+    if (gDropdownNumItems == 0) {
+        _dropdown_num_columns = 1;
+        _dropdown_num_rows = 0;
+    }
+    else {
```

Comment:
@Gymnasiast 
I'm looking through the file and it seems this file is following the format: 
```
if (expression) {
    statement;
} else {
    statement;
}
```

The only exception where braces are on their own lines (disreguarding the void function brackets), is the `for` loop in the bottom `window_dropdown_show_colour`,Should I just go ahead and move the `else {` onto the same line as `}` ?

Unless we are about to switch to having control brackets all on their own line for some reason. (Personally speaking, I prefer my way because having the end bracket on its own line allows you to see where the if statement ends, and if there's any `else`s that go in pair with the `if`. but that's just my personal opinion, i'll still adapt to whatever style)